mtags/src/main/scala-2/scala/meta/internal/pc/Keywords.scala

```diff
@@ -10,8 +13,22 @@ trait Keywords { this: MetalsGlobal =>
       pos: Position,
       editRange: l.Range,
       latestEnclosing: List[Tree],
-      completion: CompletionPosition
+      completion: CompletionPosition,
+      text: String
   ): List[Member] = {
+    val start = pos.start
```

Comment:
Could you put it into a def? And later w can do
`lazy val isNotInComment = checkIfNotInComment()`